### PR TITLE
ci: check broken links during pr #491

### LINKED ISSUE #491: [CI] Add broken links check
## Background && What & Why

In a large document project, broken links are very likely to occur, such as the recent issue #490. We cannot rely on people to manually discover broken links, so we should use CI to automatically check them.

## Design && How to do it

Use [lychee](https://github.com/lycheeverse/lychee-action) to find broken links.

check during:

* a new PR
* schedule: once a day on main branch

* [x] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by aFlyBird0**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have `npm run build` and `npm run serve` locally before submitting this PR
- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary

check broken links during 

* pr
* main branch daily. And submit an issue automatically if there are broken links

### Does this close any open issues?
Closes #491

### Screenshots

* actions during PR when there are broken links: https://github.com/aFlyBird0/test-devlake-website-create-issue/actions/runs/4778339515
* on main branch
  * actions: https://github.com/aFlyBird0/test-devlake-website-create-issue/actions/runs/4778402141
  * issue created by actions example: https://github.com/aFlyBird0/test-devlake-website-create-issue/issues/2

### CODE REVIEW

**Comment by aFlyBird0** [comment]
Hi Devlake Team 🙌, you could merge this pr first. Then I will try to fix all broken links in next pr.

File: .github/workflows/link-schedule.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Use Node.js 17.8.x
+        uses: actions/setup-node@main
+        with:
+          node-version: 17.8.x
+      - name: Build Docusaurus website
+        run: |
+          npm install
+          npm run build
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.5.0
```
Comment (klesh) [inline .github/workflows/link-schedule.yaml]: Oh, I don't think this would work...

Comment (aFlyBird0) [reply .github/workflows/link-schedule.yaml]: Got it, thank you. I'll use cmd line to refactor it and send a new review request after finished.

**Comment by aFlyBird0** [comment]
@klesh I have update the actions code and update the PR body. Please review it again.

**Review by klesh — DISMISSED** [review]
LGTM

**Comment by klesh** [comment]
@aFlyBird0 Seems like some checks didn't pass, was it expected?

**Comment by aFlyBird0** [comment]
> @aFlyBird0 Seems like some checks didn't pass, was it expected?

Yes, I'll fix broken links in next PR. This PR is only for actions.

**Comment by aFlyBird0** [comment]
@klesh I have fixed almost all broken links: https://github.com/aFlyBird0/incubator-devlake-website/actions/runs/4794001248 

except these two links:

The first link is a GitHub account that seems to have been deleted. I haven't found the current address for the second link yet.

image

image

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**